### PR TITLE
Coinbase not updating

### PR DESCRIPTION
**Pull request by wolffan**
Custom Version of Snapkit for beta3
Custom Version of TrustKit for Xcode9beta3

### CODE REVIEW

File: BalanceOpen/Data Model/API Models/CoinbaseAccount.swift [hunk 1]
```diff
@@ -12,6 +12,8 @@ fileprivate var decimalFormatter: NumberFormatter = {
     let formatter = NumberFormatter()
     formatter.generatesDecimalNumbers = true
     formatter.numberStyle = .decimal
+    formatter.allowsFloats = true
+    formatter.locale = Locale(identifier: "en_US")
```
Comment (wolffan) [inline BalanceOpen/Data Model/API Models/CoinbaseAccount.swift]: this is part of a different error related to the app running on non uk/us locale

File: BalanceOpen/Data Model/Static & Singletons/CertValidator.m [hunk 2]
```diff
@@ -9,10 +9,15 @@
 #import "CertValidator.h"
 #import <TrustKit/TrustKit.h>
 
+@interface CertValidator ()
+    @property (nonatomic, readonly, strong) TSKPinningValidator* pinningValidator;
```
Comment (wolffan) [inline BalanceOpen/Data Model/Static & Singletons/CertValidator.m]: related with Xode 9 issues

**Comment by wolffan** [comment]
Thanks for this sweet fast review @reddavis 
BTW you then had your token expired? Have you re-re launched?

**Comment by reddavis** [comment]
I did re-re launch and have just launched again and Balance opens and syncs fine

**Comment by wolffan** [comment]
Thanks, o I forgot we're using a custom branch of Trustkit and sourcekit (i'll check if they have merged my changes)

**Comment by wolffan** [comment]
Ok I'll update TrustKit since they have merged my changes
Sadly we'll have to wait on SourceKit

**Pull request merged**